lib/matplotlib/collections.py

```diff
@@ -16,6 +16,7 @@
 import matplotlib as mpl
 from . import (_path, artist, cbook, cm, colors as mcolors, docstring,
                lines as mlines, path as mpath, transforms)
+from .rcsetup import validate_joinstyle, validate_capstyle
```

Comment:
Import the module, not the function (otherwise people will start relying on the function being available in this module which will break havoc if we ever try to move it).